### DISCUSSION TITLE
1-4-1 inline requester/provider stations

### DISCUSSION THREAD

**Discussion by Frerduro** [Opinionated-Blueprints/10-Books-Full-of-Rails#91] [Ideas]
I would be nice to have 1-4-1 stations the current 2-4-0 stations don't work for me.

**Comment by Bocian-1** [comment]
It sure would. But I'm not making them. The current roster is already a huge pain to update and often requires multiple passes to correct inevitable mistakes.

**Reply by brandon-gradient** [reply]
Not saying I will build it but if I did make something like a 1-4-1 inline that worked with this book of books it would be nice to have a dedicated area to share some of those things. Though, I'm more likely to make a 1-4-0 inline, or possibly a 1-1S1L-0 (for uranium)

**Comment by thomasdeniau** [comment]
1-4-0 inline would be awesome!

**Reply by rpdelaney** [reply]
Pull requests are welcome. :)
